Documentation/dl_build.md

```diff
@@ -64,4 +67,3 @@ If OK is printed, then etcd is working!
 [go]: https://golang.org/doc/install
 [build-script]: ../build
 [cmd-directory]: ../cmd
-
```

Comment:
It works even otherwise as given as `[Example hardware configuration](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#example-hardware-configurations)`